HTTPServerProject/WriteStreams.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Net;
+using System.Net.Sockets;
+using System.IO;
+using System.Collections.Generic;
+
+namespace HTTPServerProject.WriteStream
+{
+    public interface IWriteStreams
+    {
+        void WriteLine(string str = default!);
+        void Flush();
+        void Close();
+    }
+
+    public class WriteStreams : IWriteStreams
+    {
+        Stream rStream = new MemoryStream();
+        StreamWriter writer;
+
+        public WriteStreams(Stream stream)
+        {
+            rStream = stream;
+            writer = new StreamWriter(stream);
+        }
+
+        public void WriteLine(string str = default!)
```

Comment:
Why provide a default implementation? Also I think you could just call `WriteLine(str)` no matter what.

Reply:
My thinking is that calling `WriteLine()` will create a blank CRLF line to separate the headers and body.